components/post-primary-term/index.js

```diff
@@ -0,0 +1,47 @@
+import { __ } from '@wordpress/i18n';
+import PropTypes from 'prop-types';
+import { usePrimaryTerm } from '../../hooks';
+
+export const PostPrimaryTerm = (props) => {
+	const {
+		taxonomyName = 'category',
+		placeholder = __('Select a term', 'tenup'),
+		isLink = true,
```

Comment:
Might add `category` as a defaultProps here and remove the defaults here too as suggested elsewhere.

Reply:
fixed in [27c0ffc](https://github.com/10up/block-components/pull/178/commits/27c0ffce7b057bbdc29edb3607fd4eaffefdde41)